File: HackerRank-10-Days-Statistics/Day 4/Day_4_Geometric_Distribution_I.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import sys
[a,b]=sys.stdin.readlines()
num, den=map(int,a.strip().split())
n=int(b)

p_d=num/den
print(round((((1-p_d)**(n-1)) * p_d),3))

File: HackerRank-10-Days-Statistics/Day 4/Day_4_Geometric_Distribution_I.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include<bits/stdc++.h>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */  
    float num, den;
    int n;
    cin>>num>>den>>n;
    float p_d=num/den;
    
    cout<<fixed<<setprecision(3)<<pow(1-p_d,n-1)*p_d<<endl; 
    return 0;
}
